validator-api/src/config/mod.rs

```diff
@@ -178,11 +180,23 @@ impl Default for TopologyCacher {
     }
 }
 
+fn strip_trailing_newline(input: &str) -> String {
+    input
+        .strip_suffix("\r\n")
```

Comment:
`.trim()` will do exactly that - it will remove all leading and trailing whitespaces (so it includes newlines)